source/clear-linux/guides/maintenance/mixin.rst

```diff
@@ -1,101 +1,116 @@
-version of |CL|.
+The mixin tool, a light wrapper for :ref:`mixer`, is used to customize your
+installation of |CL-ATTR| by *side loading* a custom bundle. This is known
+as a *mixin*. Creating a mixin is useful when you need to add custom or 3rd
+party content but want to keep on the upstream update cycle. This guide covers
+the workflow of creating your own mixin using the mixin tool.
 
-The first method is to use the :ref:`mixer tool<mixer>` to create your own
-|CL| image and add your bundles to it.  Mixing your own |CL| image can
-give you great control and flexibility; however, you must act as an
-:abbr:`OSV (Operating System Vendor)` and maintain your releases and
-updates because you have forked from upstream.
+If you need a greater degree of customization than provided by mixin, use our
+:ref:`mixer tool<mixer>`.
 
-The second method is to use the :command:`mixin` tool, which also
-makes use of mixer to create custom bundles that you can add to your
-upstream |CL| system.  This  simpler method provides a “light” forking from
-upstream, which means you can continue to get upstream bundles and updates.
-If needed, you can easily revert your system back to the upstream version.
+Prerequisites
+*************
 
-This guide shows you how to accomplish the second method by following these
-steps:
+To start working with the mixin tool, you will need a recent image of |CL| with
+the `mixer` bundle installed. Before you install any new packages, update your
+OS with the following command:
 
-#. Set up the workspace.
-#. Copy your custom RPM package to the workspace.
-#. Create a bundle with your custom RPM package.
-#. Migrate your |CL| system to your custom mix.
-#. Add your custom bundle to your system.
-#. Optional: Revert your system back to 100% upstream.
+.. code-block:: bash
 
-Set up the workspace
-********************
+   sudo swupd update
 
-#. Install the mixer bundle to enable mixer.
+If the bundle is not yet installed, you can add it with the :command:`swupd bundle-add`
+command as follows:
 
-   .. code-block:: console
+.. code-block:: bash
 
-      $ sudo swupd bundle-add mixer
+   sudo swupd bundle-add mixer
 
-#. Create the workspace.
+Mixin workflow
+**************
 
-   .. code-block:: console
+The workflow to create your own mixin is outlined below.
 
-      $ sudo mkdir -p /usr/share/mix/local-rpms
+#. :ref:`create-workspace-mixin`
+#. :ref:`create-rpm`
+#. :ref:`copy-custom-rpm`
+#. :ref:`create-bundle`
+#. :ref:`migrate-mix`
+#. :ref:`add-bundle-to-system`
+#. :ref:`revert-to-upstream`
 
-Copy your custom RPM package to the workspace
-*********************************************
+.. _create-workspace-mixin:
 
-.. note::
+Create a workspace
+==================
+
+Use the following command to create an empty directory in your |CL| image to
+use as a workspace for mixing:
+
+.. code-block:: bash
+
+   sudo mkdir -p /usr/share/mix/local-rpms
+
+.. _create-rpm:
+
+Create or locate RPMs for mixin
```

Comment:
In context of the include used here, I recommend removing this header, "Create or locate RPMs..."

Reply:
Per our chat, I will add some clarifying text re local vs 3rd party/remote RPMs.